### DISCUSSION TITLE
Alex changed his username to uNetworkingAB

### DISCUSSION THREAD

**Discussion by knight10-ux** [uNetworking/uWebSockets.js#789] [Q&A] [answered]
~~It is an important issue. Are these repos have a maintainer or is he gone for good? Has he been hacked? Why deleting an account without saying anything?~~

My bad sorry, I don't know how I could miss latest commits. It's just that I confused user account uNetworkingAB with uNetworking organization account.

**Comment by dvergeylen** [comment, marked as answer]
>Alex Huntman deleted his github account?

He actually created a new one, but associated the same e-mail address: https://github.com/uNetworkingAB.
You can see the commits are now associated with this profile because of the same e-mail address in use.

**Reply by knight10-ux** [reply]
uffff thank you!
